04week/loop.js

```diff
@@ -0,0 +1,72 @@
+// for loop
+// Use a for loop to console.log each item in the array carsInReverse.
+
+const carsInReverse = ["Ford", "Chevy", "Toyota"];
+carsInReverse.forEach(function(item, index, array) {
+  console.log(item, index);
+});
+
+// for...in loop
+// Create an object (an array with keys and values) called persons with the following data:
+// firstName: "Jane"
+// lastName: "Doe"
+// birthDate: "Jan 5, 1925"
+// gender: "female"
+// Use a for...in loop to console.log each key.
+// Then use a for...in loop and if state to console.log the value associated with the key birthDate.
+
+const persons = {
+  firstName: "Jane",
+  lastName: "Doe",
+  birthDate: "Jan 5, 1925",
+  gender: "female",
+};
+console.log(Object.keys(persons));
+
+for (let birthDate in persons) {
+  person1 = persons.birthDate;
```

Comment:
you need to use an if statement to check if the key birthdate is === to the key that is currently being looped through.